spec/remote_record_spec.rb

```diff
@@ -36,6 +36,37 @@ def client
     initialize_reference
   end
 
+  describe '#skip_fetching' do
+    before do
+      skip 'relies on database'
```

Comment:
I don't quite understand what this is doing

Reply:
These tests are necessary for that scope, but we can't put them to work just yet because we don't have a database backing the tests.